bridges/WikipediaBridge.php

```diff
@@ -107,9 +107,18 @@ private function AddElementGeneric($element, $fullArticle){
 		// Clean the bottom of the featured article
 		$element->find('div', -1)->outertext = '';
 
+		// The title and URI of the article is best defined in an anchor containint the string '...' ('full article ...')
+		$target = $element->find('p/a', 0); // We'll use the first anchor as fallback
+		foreach($element->find('//a') as $anchor){
+			if(strpos($anchor->innertext, '...') !== false){
```

Comment:
>Anyway, off the top of my head, how about something like this?

Nice idea, looks very clean and is much easier to fix in case things break. I like it :+1: 

>I can also toss it into a PR myself later if you prefer.

Please do. You have thought that through, so I think it would be best that way :smile: 

Also:

>and I'm sure it'd break down for Dutch and Italian Wikipedia at some point as well.

There is no support for Dutch and Italian Wikipedia as of now, feel free to add it though :innocent: